python/flink_agents/api/runner_context.py

```diff
@@ -65,6 +65,11 @@ def get_action_config(self) -> Dict[str, Any]:
           The configuration of the action executed.
         """
 
+    @property
+    def action_config(self) -> Dict[str, Any]:
```

Comment:
Instead of introducing new property methods, can we rename the current get method and make it both abstract and property method?